var/spack/repos/builtin/packages/detray/package.py

```diff
@@ -19,6 +19,7 @@ class Detray(CMakePackage):
 
     license("MPL-2.0", checked_by="stephenswat")
 
+    version("0.95.0", sha256="86cc981eb0105143b971acea3544b9a668326e1027f317d77cf76918f766e7c4")
```

Comment:
FYI: `cmake_minimum_required(VERSION 3.21)` per README.md and some lower-level CMakeLists.txt, but not top-level CMakeLists.txt. Does that need to be propagated here for the entire package, or dependent on variants?

Reply:
Well spotted. I've hadded the CMake dependency update.